### PR TITLE
#4059, drag drop error

### PR DESCRIPTION
**Pull request by jagjeetst**
## Issue

File manager overlay within Edit mode supports upload of files by dragging them onto the window but it doesn’t work reliably. Progress bar appears but it never completes. #4059

## Description

Fixed uploading of images by dragging them in upload area where loader was keeps on showing.

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -536,9 +540,15 @@ function uploadFiles(files) {
...
 
+  if (currentFolderId == null && currentAppId == null) {
+    currentOrganisationId = currentOrgId;
+  } else {
+    currentOrganisationId = null;
+  }
   Fliplet.Media.Files.upload({
     folderId: currentFolderId,
     appId: currentAppId,
+    organizationId : currentOrganisationId,
```
Comment (squallstar) [inline js/interface.js]: please check our code guidelines, there should not be a space after the property name

File: widget.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "File Manager",
-  "package": "com.fliplet.file-manager",
+  "package": "com.fliplet.file-manager.org-drag-drop",
```
Comment (squallstar) [inline widget.json]: This should not have been committed

File: js/interface.js (hunk 1 above)
Comment (jagjeetst) [reply js/interface.js]: Fixed.

File: widget.json (hunk 2 above)
Comment (jagjeetst) [reply widget.json]: Fixed..

**Pull request closed without merging**